aws/resource_aws_msk_cluster_test.go

```diff
@@ -403,6 +414,7 @@ func TestAccAWSMskCluster_NumberOfBrokerNodes(t *testing.T) {
 					testAccCheckMskClusterNotRecreated(&cluster1, &cluster2),
 					resource.TestCheckResourceAttr(resourceName, "bootstrap_brokers", ""),
 					resource.TestMatchResourceAttr(resourceName, "bootstrap_brokers_tls", regexp.MustCompile(`^(([-\w]+\.){1,}[\w]+:\d+,){2,}([-\w]+\.){1,}[\w]+:\d+$`)),
+					resource.TestCheckResourceAttr(resourceName, "broker_client_vpc_ip_addresses.#", "6"),
```

Comment:
I could use some assistance with this test which is failing. What I suspect is happening is that `broker_client_vpc_ip_addresses` is set to 3 when the cluster is initially created, but when the cluster is expanded to 6 brokers that `broker_client_vpc_ip_addresses` is not being re-calculated and updated. Please let me know if any tips can be provided to debug this type of issue.

Reply:
I made adjustments to the unit test in https://github.com/terraform-providers/terraform-provider-aws/commit/a1ec11fe80bf7fe0d01b14cb5c6b26c808bd598a so that broker_client_vpc_ip_addresses is obtained a second time after the cluster has expanded. Please let me know if there is a more canonical method which should be used for a test of this kind.